bundles/io/org.eclipse.smarthome.io.transport.mdns/src/main/java/org/eclipse/smarthome/io/transport/mdns/internal/MDNSClientImpl.java

```diff
@@ -19,20 +19,25 @@
 import java.time.Duration;
 import java.util.Enumeration;
 import java.util.HashSet;
+import java.util.List;
```

Comment:
This MDNSClientImpl.java file was not part of this PR before, did you add it by intention?

Reply:
Thanks, that's should be next PR.